src/components/VizbeeMiniCastBar/helpers.js

```diff
@@ -0,0 +1,69 @@
+
+/**
+   * Gets the miniplayer title to be displayed
+   *
+   * @returns {string} title to display
+   */
+export const getTitleFromVideoInfo = ({title, castingTo}) => {
+    if (title && title?.length) {
+      return title;
+    }
+
+    if (castingTo) {
+      return `Casting to ${castingTo}`;
+    } else {
+      return "";
+    }
+  };
+
+  /**
+   * Gets the miniplayer subtitle to be displayed
+   *
+   * @returns {string} subtitle to display
+   */
+  export const getSubTitleFromVideoInfo = ({title, subTitle, castingTo}) => {
+    if (subTitle && subTitle?.length) {
+      return subTitle;
+    }
+
+    if (title && title?.length) {
+      return `Casting to ${castingTo}`;
+    }
+
+    return "";
+  };
+
+/**
+   * Gets the default color for the apperance
+   *
+   * @returns {Object} default color object
+   */
+  export function getApperanceColor(isDark) {
```

Comment:
is this typo? `getApperanceColor` --> `getAppearanceColor`

Reply:
Done